### PR TITLE
Milestone 2: Allow print to use the @ operator

### PR DESCRIPTION
**Pull request by NicholasRM**
- Allow accessing of specific data members of structures or indices of lists, tuples, and strings
- Correctly handle malformed input (i.e. `@foo`, `bar@` `bar@@foo`)
- Prohibit indexing into structures or accessing data members on anything other than a structure

### CODE REVIEW

File: docs/MAD.txt
Comment (lutzhamel) [inline docs/MAD.txt:1]: You need to generate a .rs file and review if the text looks acceptable in the restructured text file, I use the python module restview in order to see this locally on my computer. (https://pypi.org/project/restview/)

Comment (NicholasRM) [reply docs/MAD.txt:1]: Thanks for letting me know. The .rst files have been regenerated and look correct

**Comment by lutzhamel** [comment]
@NicholasRM Does my comment on Discord about the fact that term2string handles objects, lists, and strings change anything in this PR?

**Review by lutzhamel — APPROVED** [review]
Looks good! Nice work with the printing part.  Feel free to merge this.

**Pull request merged**